cypress/integration/page.spec.js

```diff
@@ -1,7 +1,90 @@
-// WRITE TESTS HERE
+const page = {
+  showClock() {
```

Comment:
```suggestion
  getShowClockButton() {
```
інакше виглядає, що метод покаже годинник, а не поверне елемент